mitosheet/src/mito/Mito.tsx

```diff
@@ -1003,12 +1003,35 @@ export const Mito = (props: MitoProps): JSX.Element => {
         'mito-taskpane-container-narrow': narrowTaskpaneOpen,
     })
 
+    const [resizingTaskpane, setResizingTaskpane] = useState(false);
+
     return (
         <div 
             className="mito-container" 
             data-jp-suppress-context-menu 
             ref={mitoContainerRef} 
             tabIndex={0}
+            onMouseMove={(event) => {
+                if (resizingTaskpane) {
+                    event.preventDefault();
+                    event.stopPropagation();
+                    const { clientX } = event;
+                    const rawPanelWidth = document.body.clientWidth - clientX;
+                    const panelWidth = Math.max(
```

Comment:
1. Can we name this taskpaneWidth? (and other variables above)
2. Also, I think let's clamp this with max and min to the current small and large taskpane sizes -- as this is what we have tested the UI to so far!

Reply:
Done!